styleguide/components/31-molecules/pagination/_pagination.scss

```diff
@@ -1,84 +1,130 @@
 .pager {
 
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
   ul {
-    margin: 0;
+    @include tablet {
+      justify-content: flex-start;
+      flex-basis: auto;
+    }
+
+    display: flex;
+    flex: 1;
+    flex-basis: 100%;
+    justify-content: center;
+    align-items: center;
+    margin: 0 1rem 0 0;
     list-style: none;
 
     li {
       @include theme('color', 'color-primary', 'link-color');
 
-      display: inline-block;
+      display: flex;
```

Comment:
Why does this li need display: flex; on it here?

Reply:
I believe it was to get everything aligned vertically.